README.md

```diff
@@ -347,3 +347,15 @@ function updateReport() {
     // ...
 }
 ```
+
+## [X] Create debug flag
+
+Add the "debug" flag if you want the created interactions to be displayed in the console.
+
+In **pacts.config.js**:
+
+```js
+module.exports = {
+    debug: true,
```

Comment:
With debug flag I would rather see whole generated file, not only interactions. It would be much easier to debug in CI.

Reply:
Can't we just inspect the generated pact to achieve the same?

I think that debug mode should log much more that just generated pact/interactions. I imagine infos like which files were analyzed, which functions, classes and interfaces were read etc

Reply:
I've renamed that flag into "verbose" to log all details about some interaction (not to print all implementation details).

Maybe in the future, we will add some more complex debugging functionality.